cg/meta/store/microsalt.py

```diff
@@ -0,0 +1,182 @@
+""" MIP specific functionality for storing files in Houskeeper """
+import datetime as dt
+import logging
+from pathlib import Path
+import ruamel.yaml
+
+from cg.constants import HK_TAGS, MICROSALT_TAGS
+from cg.exc import (
+    BundleAlreadyAddedError,
+    AnalysisDuplicationError,
+)
+
+DUMMY_MICROSALT_CASE = "dummymicrobe"
+LOG = logging.getLogger(__name__)
+
+
+def gather_files_and_bundle_in_housekeeper(config_stream, hk_api, status):
+    """Function to gather files and bundle in housekeeper"""
+    bundle_data = add_microbial_analysis(config_stream)
+
+    results = hk_api.add_bundle(bundle_data)
+    if results is None:
+        raise BundleAlreadyAddedError("bundle already added")
+
+    bundle_obj, version_obj = results
+
+    # microbial_order_obj = status.microbial_order(bundle_obj.name)
+    microbial_order_obj = status.microbial_order_by_ticket(bundle_obj.name)
+
+    new_analysis = add_new_analysis(bundle_data, microbial_order_obj, status, version_obj)
+    version_date = version_obj.created_at.date()
+
+    LOG.info("new bundle added: %s, version %s", bundle_obj.name, version_date)
+    hk_api.include(version_obj)
+    hk_api.add_commit(bundle_obj, version_obj)
+
+    return new_analysis
+
+
+def add_microbial_analysis(config_stream):
+    """Gather information from microSALT analysis to store."""
+
+    deliverables = ruamel.yaml.safe_load(config_stream)
+    new_bundle = build_microbial_bundle(deliverables)
+
+    return new_bundle
+
+
+def build_microbial_bundle(deliverables: dict) -> dict:
+    """Create a new bundle to store in Housekeeper"""
+
+    project_name = _get_microbial_name(deliverables)
+    files = microbial_deliverables_files(deliverables)
+
+    data = {
+        "name": str(project_name),
+        "created": _get_date_from_results_path(deliverables, project_name),
+        "files": files,
+    }
+    return data
+
+
+def microbial_deliverables_files(deliverables: dict) -> list:
+    """Get all deliverable files from the pipeline"""
+
+    pipeline_tags = HK_TAGS["microsalt"]
+    analysis_type_tags = MICROSALT_TAGS
+    files = parse_microbial_files(deliverables, pipeline_tags, analysis_type_tags)
+
+    return files
+
+
+def parse_microbial_files(
```

Comment:
Would it be possible to reuse the `parse_files` in cg/meta/store/base.py. I think we should not duplicates unless we expect them to diverge fast